pkg/config/system_probe.go

```diff
@@ -67,7 +67,7 @@ func setupSystemProbe(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "collect_dns_stats"), true, "DD_COLLECT_DNS_STATS")
 	cfg.BindEnvAndSetDefault(join(spNS, "collect_local_dns"), false, "DD_COLLECT_LOCAL_DNS")
 	cfg.BindEnvAndSetDefault(join(spNS, "collect_dns_domains"), false, "DD_COLLECT_DNS_DOMAINS")
-	cfg.BindEnvAndSetDefault(join(spNS, "max_dns_stats"), 10000)
+	cfg.BindEnvAndSetDefault(join(spNS, "max_dns_stats"), 20000, "DD_MAX_DNS_STATS")
```

Comment:
Note that all config values do have env vars bound by default now, they are just a bit verbose: `DD_SYSTEM_PROBE_CONFIG_MAX_DNS_STATS`

Reply:
Oh. Did not know that :(. 🙇 . Will remove it then.

Reply:
@brycekahle Updated the PR to get rid of the explicit environment variable.